app/workflows/hearing_request_case_distributor.rb

```diff
@@ -35,7 +35,20 @@ def appeals_to_distribute
   end
 
   def create_judge_assign_task_for_appeal(appeal)
-    JudgeAssignTaskCreator.new(appeal: appeal, judge: distribution.judge).call
+    judge_assign_task_creator = JudgeAssignTaskCreator.new(appeal: appeal, judge: distribution.judge)
+    current_judge_assign_tasks = appeal.tasks.select { |task| task.type == JudgeAssignTask.name && task.open? }
+    if current_judge_assign_tasks.empty?
```

Comment:
```suggestion
    if current_judge_assign_tasks.blank?
```
See https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html